src/components/search-form.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { useStaticQuery, graphql } from 'gatsby';
 import { navigate } from '@reach/router';
+import search from '../../node_modules/uswds/dist/img/usa-icons-bg/search--white.svg';
```

Comment:
You could pull in the svg from the package directly.

```suggestion
import search from 'uswds/src/img/usa-icons-bg/search--white.svg';
```

Reply:
@apburnes Not sure why but the other components were using that relative path syntax, but I went ahead and updated them as you suggested for consistency: https://github.com/18F/federalist-uswds-gatsby/pull/474/commits/bf9a86cfb6ee7f94afd346d45b28616604b7701f

Reply:
💯